frontend/dashboard/pages/OrgContentLibrary/OrgContentLibrary.tsx

```diff
@@ -29,51 +29,64 @@ export function OrgContentLibrary(): ReactElement {
   const selectedContext = useSelectedContext();
 
   return isOrg(selectedContext) ? (
-    <OrgContentLibraryWithContext />
+    <OrgContentLibraryWithOrg orgName={selectedContext} />
   ) : (
     <ContextWithoutLibraryAccess />
   );
 }
 
-function OrgContentLibraryWithContext(): ReactElement {
-  const { t } = useTranslation();
-  const selectedContext = useSelectedContext();
-  const orgRepoName = useOrgRepoName();
-
-  const { data: repoStatus } = useRepoStatusQuery(selectedContext, orgRepoName);
-
-  useListenToMergeConflictInRepo(selectedContext, orgRepoName);
+type OrgContentLibraryWithOrgProps = {
+  orgName: string;
+};
 
-  const { data: codeListsResponse, status: codeListResponseStatus } =
-    useOrgCodeListsQuery(selectedContext);
+function OrgContentLibraryWithOrg({ orgName }: OrgContentLibraryWithOrgProps): ReactElement {
+  const orgRepoName = useOrgRepoName();
+  const { data: repoStatus } = useRepoStatusQuery(orgName, orgRepoName);
+  useListenToMergeConflictInRepo(orgName, orgRepoName);
 
   if (repoStatus?.hasMergeConflict) {
-    return <MergeConflictWarning owner={selectedContext} repoName={orgRepoName} />;
+    return <MergeConflictWarning owner={orgName} repoName={orgRepoName} />;
+  } else {
+    return <MergeableOrgContentLibrary orgName={orgName} />;
   }
+}
+
+type MergeableOrgContentLibraryProps = {
+  orgName: string;
+};
+
+function MergeableOrgContentLibrary({ orgName }: MergeableOrgContentLibraryProps): ReactElement {
+  const { t } = useTranslation();
+  const { data: codeListsResponse, status: codeListResponseStatus } = useOrgCodeListsQuery(orgName);
 
   switch (codeListResponseStatus) {
     case 'pending':
       return <StudioPageSpinner spinnerTitle={t('general.loading')} />;
     case 'error':
       return <StudioPageError message={t('dashboard.org_library.fetch_error')} />;
     case 'success':
-      return <OrgContentLibraryWithContextAndData codeListsDataList={codeListsResponse} />;
+      return (
+        <OrgContentLibraryWithContextAndData
+          codeListsDataList={codeListsResponse}
+          orgName={orgName}
+        />
+      );
   }
 }
 
 type OrgContentLibraryWithContextAndDataProps = {
   codeListsDataList: CodeListData[];
+  orgName: string;
 };
 
 function OrgContentLibraryWithContextAndData({
   codeListsDataList,
```

Comment:
Do you think this is a clearer variable name?
```suggestion
        <OrgContentLibraryWithContextAndData
          codeListDataList={codeListsResponse}
          orgName={orgName}
        />
      );
  }
}

type OrgContentLibraryWithContextAndDataProps = {
  codeListDataList: CodeListData[];
  orgName: string;
};

function OrgContentLibraryWithContextAndData({
  codeListDataList,
```

Reply:
Absolutely. This is fixed now.